16-using-svn.md

```diff
@@ -149,5 +149,25 @@ This will open your editor, where you should enter a message describing the fold
 You can checkout this repository and begin playing with it using the commands above.
 
 ```bash
-$ svn co svn+ssh://svn.cern.ch/reps/lhcbdocs/Users/$USERNAME/TestFolder
+svn co svn+ssh://svn.cern.ch/reps/lhcbdocs/Users/$USERNAME/TestFolder
 ```
+
+
+> ## Using SVN outside the lxplus cluster {.callout}
+> Sometimes you may want to `getpack` and/or `commit` from other places, e.g. your local machine, the online cluster...
```

Comment:
Would change it to "If you want to getpack .... from other places". I dislike the "sometimes", "maybe", "potentially" kind of style